README.adoc

```diff
@@ -163,21 +161,21 @@ Generally you will not need to use `spring-boot-loader` directly, but instead wo
 
 
 === spring-boot-devtools
-The spring-boot-devtools module provides additional development-time features such as automatic restarts, for a smoother application development experience.
+The spring-boot-devtools module provides additional development-time features such as automatic restarts for a smoother application development experience.
```

Comment:
"such as automatic restarts" should have commas on both ends. (It's parenthetical.)